docs/user-guide/members/README.md

```diff
@@ -74,19 +74,35 @@ Find out [here](../../getting-started/invite-members/#add-existing-users) how to
 
 To change the role of a member within a project, select the corresponding project and open the Members module.
 
-To edit an existing member of a project, click the **Edit icon** in the list next to the member on the right. Choose the new role and press the blue **Change button** so save your changes.
+To edit an existing member of a project, click the **Icon with three dots** in the list next to the member on the right and select **Manage roles**. Choose the new role and press the green **Change ** button so save your changes.
 
 ![Edit project members in OpenProject](edit-project-member.png)
 
 
 
 ## Remove members
 
-To remove members from a project, [select the project](../../getting-started/projects/#open-an-existing-project) for which you want to remove the members. In the project menu on the left, select the **Members** module. In the members list, click the delete icon at the right end of the row with the corresponding member name.
+To remove members from a project, [select the project](../../getting-started/projects/#open-an-existing-project) for which you want to remove the members. In the project menu on the left, select the **Members** module. In the members list, click the three dot icon at the right end of the row with the corresponding member name and select **Remove member**.
 
 ![Remove project members in OpenProject](delete-project-member.png)
 
+You will be asked to confirm your decision. 
 
+![Confirm removing a user from a project in OpenProject](confirm-user-deletion.png)
+
+If the project member you are removing has shared work packages, you will also be asked whether these sharing rights also need to be removed.
+
+![Confirm removing a user with shared work packages from a project in OpenProject](openproject_user_guide_confirm_user_removal_with_shares.png)
+
+## Revoke sharing privileges
+
+If a work package has been [shared](../work-packages/share-work-packages), you may need to revoke sharing privileges at a later stage in the project. To do that select the **three dot icon** at the right end of the row with the corresponding member name and select **Revoke work package shares**. You can also choose the **View shared work packages** option to see the list of all shared work packages.
+
+![Remove sharing privileges in OpenProject](openproject_user_guide_members_remove_work_package_shares.png)
+
+> Note: it is not possible to revoke sharing rights of project members, if they are a part of a project only as members of a group. In this case you can either remove a single member from the group (and then remove the rights), or revoke the shared privileges from the entire group.
```

Comment:
I find this a bit confusing. If this disclaimer is about revoking work package shares I will phrase it a bit different:

`Note: the revoking action will only affect the individually work packages shares. All those shares with the user via a group will not be revoked. To revoke those group shares you can either remove the member from the group or revoke the privileges from the entire group.`

With that we could delete the sentence bellow.